types/node-red/index.d.ts

```diff
@@ -1,13 +1,12 @@
-// Type definitions for node-red 1.2
+// Type definitions for node-red 1.3
 // Project: https://github.com/node-red/node-red/tree/master/packages/node_modules/node-red, https://nodered.org/
 // Definitions by: Anders E. Andersen <https://github.com/andersea>
 //                 Thomas B. Mørch <https://github.com/tbowmo>
 //                 Bernardo Belchior <https://github.com/bernardobelchior>
 //                 Alex Kaul <https://github.com/alexk111>
+//                 Tadeusz Wyrzykowski <https://github.com/Shaquu>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// Minimum TypeScript Version: 3.1
-
-// tslint:disable:no-empty-interface
```

Comment:
Except for that line which is reverted in the other commit, all essence of PR is in that first commit.